projects/eslint-plugin-experience/rules/prefer-deep-imports.js

```diff
@@ -0,0 +1,98 @@
+const fs = require('fs');
+const glob = require('glob');
+
+const MESSAGE_ID = `prefer-deep-imports`;
+const ERROR_MESSAGE = `Import via root level entry point are prohibited for this package`;
+
+module.exports = {
+    create(context) {
+        const {importFilter} = context.options[0] || {};
+
+        return {
+            [`ImportDeclaration[source.value=${importFilter}]`](
+                importDeclaration,
+                ...rest
+            ) {
+                const importedEntities = importDeclaration.specifiers;
+                const packageName = importDeclaration.source.value;
+
+                context.report({
+                    fix: fixer => {
+                        const allTsFiles = glob.globSync(
+                            `node_modules/${packageName}/**/*.ts`,
+                            {
+                                ignore: {
+                                    ignored: p => /\.spec.ts$/.test(p.name),
```

Comment:
what about `cy.ts`

Reply:
fixed in https://github.com/taiga-family/linters/pull/438/commits/4ade942bca7a61154914f165a42281bc3847b645